Gems/Atom/RPI/Code/Source/RPI.Reflect/Material/ShaderCollection.cpp

```diff
@@ -51,8 +43,13 @@ namespace AZ
                 }
                 else
                 {
+                    // No worries, eventually the Material::Init will end up
+                    // calling InitializeShaderOptionGroup() and @m_shaderOptionGroup
+                    // will get proper data.
                     shaderVariantReference->m_shaderOptionGroup = {};
+                    shaderVariantReference->m_shaderAsset.QueueLoad(); // Not necessaru to call QueueLoad, but doesn't hurt either.
```

Comment:
nit: if you're really sure this line is unnecessary, just take it out. The above comment suffices.

Reply:
I'm not 100% sure (more like 99%), but I tested without it and it works, yet left it anyways because it is not hurting.